styles.css

```diff
@@ -219,12 +214,6 @@ h6 {
 
 }
 
```

Comment:
Thanks for fixing this, interesting how it was only causing one of the error messages to have changed styling